src/test/java/api/loans/scenarios/CheckoutWithRequestScenarioTests.java

```diff
@@ -121,6 +139,14 @@ public void checkingOutWithHoldRequestAppliesAlternatePeriodAndScheduledForFixed
     final IndividualResource james = usersFixture.james();
     final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
 
+    DateTime now = new DateTime(DateTimeZone.UTC)
```

Comment:
Why is the variable for the frozen time named `now` in this test and `fakeNow` in the other test?

Reply:
This changes do not required, as the test is fixed on master.